packages/libs/react-ui/src/components/Link/Link.stories.tsx

```diff
@@ -68,7 +68,6 @@ type LinkStory = StoryObj<ILinkProps>;
 export const _Button: LinkStory = {
   args: {
     children: 'Hello world',
-    variant: 'primary',
```

Comment:
i believe the variant arg type defined in meta is incorrect. I think there should be color and style variant?

Reply:
Link and Button have the same API, except for the default value of the variant prop. So by removing this prop you get the "transparent" variant.

Reply:
ahh I see the variant includes styles and colors. thanks!